### PR TITLE
Improve logging and site reliability checks

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Ticket Number

UCD-961

## Description of change

 - Added some logging around the app
 - Refactored the call to Appeals Service API to a handler function, with a timeout, logging etc
 - Created base handlers for 404 and 500 errors inside Express
 - Add fix in common to always `clearTimeout`

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [x] I have merged commits to avoid fixing things in this PR
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: forms-web-app/src/lib/appeals-api-wrapper.js [hunk 1]
```diff
@@ -1,35 +1,75 @@
 const fetch = require('node-fetch');
+const uuid = require('uuid');
+const { utils } = require('@pins/common');
+
 const config = require('../config');
+const parentLogger = require('./logger');
+
+async function handler(url, method = 'GET', opts = {}, headers = {}) {
```
Comment (a6software) [inline forms-web-app/src/lib/appeals-api-wrapper.js:8]: Would renaming `url` to `path` be beneficial here? 

`url` implies I can / should pass in a full url, but that seems to break line 10

File: forms-web-app/src/views/error/not-found.njk [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{% extends "layouts/main.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %},
```
Comment (a6software) [inline forms-web-app/src/views/error/not-found.njk:2]: can remove this one

File: forms-web-app/src/views/error/unhandled-exception.njk [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% extends "layouts/main.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %},
```
Comment (a6software) [inline forms-web-app/src/views/error/unhandled-exception.njk:2]: can remove this one

**Review by a6software — CHANGES_REQUESTED** [review]

File: forms-web-app/src/views/error/not-found.njk (hunk 2 above)
Comment (MrSimonEmms) [reply forms-web-app/src/views/error/not-found.njk:2]: Done - presume just the line 2 stuff?

File: forms-web-app/src/views/error/unhandled-exception.njk (hunk 3 above)
Comment (MrSimonEmms) [reply forms-web-app/src/views/error/unhandled-exception.njk:2]: Done - presume just the line 2 stuff?

File: forms-web-app/src/lib/appeals-api-wrapper.js (hunk 1 above)
Comment (MrSimonEmms) [reply forms-web-app/src/lib/appeals-api-wrapper.js:8]: Yeah, I'm easy either way with names. Will rename to `path` - the idea with this handle is that logging, authentication (if/when it comes) and routing are all handled automagically and the name of the variable makes no difference

**Review by a6software — APPROVED** [review]
LGTM 👍

**Pull request merged**